tests/ssl_test.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for the kafkit.ssl module."""
+
+import ssl
+from pathlib import Path
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
```

Comment:
Particularly for things like this, I always use the:
```
from __future__ import annotations
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from _pytest.tmpdir import TempPathFactory
```
construct because the code execution then doesn't have to rely on an import of a private-use module.